components/Question.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Header = () => {
+    return (
+      <div style={{ backgroundColor: 'white', position: 'fixed', top: 0, left: 0, right: 0, zIndex: 9999 }}>
+        <h1 style={{ textAlign: 'center', margin: 0, padding: '20px' }}>Quiz-App</h1>
+      </div>
+    );
+  };
```

Comment:
Please use styled components to add styles:

```suggestion
import styled from 'styled-components';

const StyledHeader = styled.div`
  background-color: white;
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  z-index: 9999;
`;

const Headline = styled.h1`
  text-align: center;
  margin: 0;
  padding: 20px;
`;

const Header = () => {
    return (
      <StyledHeader>
        <Headline>Quiz-App</Headline>
      </StyledHeader>
    );
  };
```